### ISSUE TITLE
Transaction created in Windows not visible on Android after sync

### ISSUE THREAD

**Issue by Ollo1** [moneymanagerex/android-money-manager-ex#1219]
Hello,
I enabled successfully synchronization between Windows and Android using OneDrive. If I create an new withdrawl in Android it will be synchronized and is visible in the Windows program. However if I perform it in the opposite way I can't see the withdrawl in Android. 
If I search for the booking within Android by e.g. Payee I can find it, however it is not visible in the cash account overview. There is no filter enabled on both apps.
Any idea?

Android Version: 2018.02.26.981
Windows: 1.4.0-Alpha 3

**Comment by MisterY** [comment]
It could be something like #1177 ?

**Comment by Ollo1** [comment]
May thanks for the prompt reply and I'm sorry that didn't find this entry, I thought I performed a good search.
The issue is similar. I noticed a difference in transactions as shown below.

```
"446"	"7"	"-1"	"1"	"Withdrawal"	"10"	""			"2"	"9"	"2018-03-13"	"-1"	"0"
"447"	"8"	"-1"	"2"	"Withdrawal"	"12.43"	"N"	""	""	"14"	"43"	"2018-03-12"	"-1"	"12.43"
```

The difference is within the 'Status' line indeed.  While the entry 446 performed in Android does not contain any status entry 447 adds an 'N'. Any idea what status this is and how to prevent it if editing in Windows?
By the way why does the windows version require a Payee value and Anroid not?

**Comment by Spoker** [comment]
At one moment, the desktop developers changed the behaviour of the statusfield and introduced statusses such as 'N', 'RN', ... I don't know the precise reasoning behind it, but it broke some other functionality (Android app, some custom reports.). These changes need to be implemented in the app. I don't think you can avoid this status, unless maybe with an older version of the desktop, but that might cause other problems.

**Comment by MisterY** [comment]
Interesting questions. :)
I thought that the "N" status was recently fixed with pull request #1195. Unfortunately, I am not using the desktop version so was not affected by that issue. The N status, as well as double-letter statuses must have been introduced recently. The double-letter status seems like a solution for reconciling two sides of a transaction separately. 
Not sure about Payee. I wonder what happens if you choose a Transfer?

**Comment by Spoker** [comment]
Just found the double digit transfer status issue: https://github.com/moneymanagerex/moneymanagerex/pull/1208
I think it is only for transfers

**Comment by Ollo1** [comment]
Thanks for responding so fast. Indeed it seems the status entries. As I don't use them on Android nor in Windows. The Windows program stores the 'N' for None while Android stores an empty string.

Independently of the type of transaction a value is needed in Windows. But that's fine I just need to be precise.

**Comment by MisterY** [comment]
Sorry, not sure why that is the case. In most finance apps I've used this type of information is not mandatory.
The same goes for status. The default for None was an empty string. Not sure why that was changed but there is probably a request in the repo.

**Comment by Ollo1** [comment]
No issue. I would thought the same. I just need something which count my balance and I don't care to whom it goes. Will you consider the new characters used in the latest Windows Alpha. I know it is Alpha stage but I was willing to test and contribute issues I found and have no problems to switch back to lastest stable.

**Comment by MisterY** [comment]
Well, as I've mentioned, the 'N' issue is, at least partially, supposed to be fixed with #1195. We should open up an issue, or rename this one to clearly identify what needs to be done and someone will pick it up when there time available to work on it.

**Comment by Ollo1** [comment]
Thanks.
